### ISSUE TITLE
Test cron for null rounds

### ISSUE THREAD

**Issue by raulk** [filecoin-project/test-vectors#124]
The tipset-class vectors have the ability to record null rounds. Null rounds trigger cron ticks. Each cron tick calls these actor methods:

![image](https://user-images.githubusercontent.com/1101242/92385165-49993b00-f109-11ea-9b78-6be6c096f627.png)

The storage power actor in turn calls the reward actor, which updates the rewards for the next tipset blocks.

We already have tests that test null rounds, but maybe we should add a specific test vector for different patterns of null rounds:

- 1 null round.
- 100 null rounds.
- 901 null rounds (beyond finality).

I guess we can assert on `reward.State` attributes? EffectiveNetworkTime, LastEpochPerReward, RewardEpochsPaid.

![image](https://user-images.githubusercontent.com/1101242/92385418-c62c1980-f109-11ea-9d57-f3836f56903e.png)
![image](https://user-images.githubusercontent.com/1101242/92385502-ec51b980-f109-11ea-8728-aa3c304156f4.png)

Basically our goal is to verify that the passing of time has been correctly reflected in the actors. We are not testing how rewards _are_ calculated, but that null rounds have been duly reflected.

Off the top of my head, an interesting flow here would be:

- 1 null round.
- 1 non-empty tipset.
- 100 null rounds.
- 1 non-empty tipset.
- 901 null rounds.
- 1 non-empty tipset.

At every non-empty tipset, we verify if the EffectiveNetworkTime, LastEpochPerReward, RewardEpochsPaid are consistent?

**Comment by raulk** [comment]
@anorth assigned to you for a sanity check and quick scoring here.
